CHANGES.md

```diff
@@ -1,16 +1,36 @@
 Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos/dcos/wiki/CHANGES.md-guidelines). Thank you!
-
-## DC/OS 1.13.6 (in development)
+## DC/OS 1.13.7 (in development)
 
 * Updated DC/OS UI to [1.13+v2.83.3](https://github.com/dcos/dcos-ui/releases/tag/1.13+v2.83.3).
 
-* Signal now sends telemetry data every 5 minutes instead of every hour. This is to align the frequency with DC/OS Enterprise.
-
 * Updated to Mesos [1.8.2-dev](https://github.com/apache/mesos/blob/c7c716dbc9ee4363ba6267591585b9984d8920b8/CHANGELOG)
 
 * Update Marathon to [1.8.232](https://github.com/mesosphere/marathon/commit/d3517aee0dfe495e2bdce34bdfc0e6f10345371b)
 
 
+### Fixed and improved
+
+* DC/OS overlay networks should be compared by-value. (DCOS_OSS-5620)
+
+* Drop labels from Lashup's kv_message_queue_overflows_total metric. (DCOS_OSS-5634)
+
+* Reserve all agent VTEP IPs upon recovering from replicated log. (DCOS_OSS-5626)
+
+* Use Golang 1.10.8 to build CockroachDB. (DCOS-61502)
+* [Mesos] Support quoted realms in WWW-Authenticate header (DCOS-61529)
```

Comment:
IIUC, this is covered by the
`Updated to Mesos [1.8.2-dev](https://github.com/apache/mesos/blob/c7c716dbc9ee4363ba6267591585b9984d8920b8/CHANGELOG)`

above, isn't it?

Reply:
I don't know I just took what was in the file before andd added it to 1.13.7.